src/components/Nurseries/index.tsx

```diff
@@ -1,5 +1,139 @@
+import { APP_PATHS } from 'src/constants';
 import strings from 'src/strings';
+import { ServerOrganization } from 'src/types/Organization';
+import { getRequestId, setRequestId } from 'src/utils/requestsId';
+import useDebounce from 'src/utils/useDebounce';
+import TfMain from '../common/TfMain';
+import PageSnackbar from '../PageSnackbar';
 
-export default function NurseriesList(): JSX.Element {
-  return <p>{strings.NURSERIES}</p>;
+type NurseriesListProps = {
+  organization: ServerOrganization;
+};
+
+const columns: TableColumnType[] = [
+  { key: 'name', name: 'Name', type: 'string' },
+  { key: 'description', name: 'Description', type: 'string' },
+];
+
+export default function NurseriesList({ organization }: NurseriesListProps): JSX.Element {
+  const { isMobile } = useDeviceInfo();
+  const history = useHistory();
+  const [temporalSearchValue, setTemporalSearchValue] = useState('');
+  const debouncedSearchTerm = useDebounce(temporalSearchValue, 250);
+  const [results, setResults] = useState<Facility[]>();
+
+  const goToNewNursery = () => {
+    const newNurseryLocation = {
+      pathname: APP_PATHS.NURSERIES_NEW,
+    };
+    history.push(newNurseryLocation);
+  };
+
+  const clearSearch = () => {
+    setTemporalSearchValue('');
+  };
+
+  const onChangeSearch = (id: string, value: unknown) => {
+    setTemporalSearchValue(value as string);
+  };
+
+  useEffect(() => {
+    const refreshSearch = async () => {
+      const params: SearchNodePayload = {
+        prefix: 'facilities',
+        fields: ['id', 'name', 'description', 'type', 'organization_id'],
+        search: {
+          operation: 'and',
+          children: [
+            {
+              operation: 'or',
+              children: [
+                { operation: 'field', field: 'name', type: 'Fuzzy', values: [debouncedSearchTerm] },
+                { operation: 'field', field: 'description', type: 'Fuzzy', values: [debouncedSearchTerm] },
+              ],
+            },
+            { operation: 'field', field: 'type', type: 'Exact', values: ['Nursery'] },
+            {
+              operation: 'field',
+              field: 'organization_id',
+              type: 'Exact',
+              values: [organization.id],
+            },
+          ],
+        },
+        count: 0,
+      };
+      const requestId = Math.random().toString();
+      setRequestId('searchNurseries', requestId);
+      const searchResults = await search(params);
+      const nurseriesResults: Facility[] = [];
+      searchResults?.forEach((result) => {
+        nurseriesResults.push({
+          id: result.id as number,
+          name: result.name as string,
+          description: result.description as string,
+          organizationId: parseInt(result.organization_id as string, 10),
+          type: result.type as 'Nursery' | 'Seed Bank' | 'Desalination' | 'Reverse Osmosis',
+          connectionState: result.connectionState as 'Not Connected' | 'Connected' | 'Configured',
```

Comment:
nit: these types should exported from the api/types using generated schema as source of truth,, and imported here. 
this list is already out dated in SeedBanks/index.tsx

Reply:
Fixed it